src/content/tutorials/observability/multi-tenancy/creating-grafana-organization/_index.md

```diff
@@ -45,3 +45,5 @@ Our operators will create this `Grafana` organization named _Giant Swarm_. It wi
 
 The Role Base Access Control (RBAC) section defines how to assign groups from your configured identity provider to `Grafana` [available roles](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/#organization-roles) (`Admin`, `Editor`, `Viewer`). For organization mapping, you can read the official Grafana [documentation](https://grafana.com/docs/grafana/next/setup-grafana/configure-security/configure-authentication/generic-oauth/#configure-role-mapping).
 Note that only the `admins` field is mandatory in this section.
+
+Thanks to our Tenant Governance, you can define any number of tenants in a Grafana Organization and update those at any time later on (i.e adding or removing tenants to an existing Grafana Organization). This also mean that deleting a Grafana Organization which tenants are also listed in other organizations won't delete those.
```

Comment:
This is already a great start, but I think we can be more precise in this. What do you think about changing it something like this: 

```suggestion
The tenant field is used to grant access to the specified tenants, but also serves as tenant governance. This means that only tenants listed in **at least one** Grafana Organisation CRD are accepted targets in the write path and can receive data. Data sent to a tenant that is not listed in any Grafana Organisation CRDs tenant field will just be dropped. 

**Warning:** Removing a tenant from **all** Grafana Organisation CRDs tenant fields also means, that you can no longer send data to that tenant! 
```